src/structures/GuildBan.js

```diff
@@ -41,8 +41,7 @@ class GuildBan extends Base {
   }
 
   /**
-   * Whether this GuildBan is a partial
-   * If the reason is not provided the value is null
+   * Whether this GuildBan is a partial. If the reason is not provided the value is null
```

Comment:
`a partial`? Wouldn't this be better worded as `partial`? @discordjs/the-big-3

Reply:
I'd also go with `partial` here.

Reply:
Yeah I agree tbh, should I change?

Reply:
Yup.